pkg/controller/podautoscaler/horizontal_test.go

```diff
@@ -111,6 +114,8 @@ func (tc *testCase) prepareTestClient(t *testing.T) *fake.Clientset {
 		MatchLabels: map[string]string{"name": podNamePrefix},
 	}
 
+	tc.Lock()
```

Comment:
I hate githube. Please add a comment by `computeCPUCurrent` that it needs to be called under the lock.